CONTRIBUTING.md

```diff
@@ -23,41 +23,27 @@ To get started, refer to the documentation for [setting up a development environ
 
 ### General contribution guidelines
 
-- A Github issue suggesting the change should be opened prior to a PR.
-- All contributions should be documented, tested and linted. Please verify that all tests and lint checks pass successfully before proposing a change.
-- In order for a pull request to be accepted, the CI (containing unit tests, e2e tests and linting) needs to succeed, in addition to approvals from two maintainers.
-- PRs should be small and solve/improve one issue at a time. If you have multiple suggestions for improvement, please open multiple PRs.
-- Every PR must bump the presidio version by changing the VERSION file to the appropriate [semver](https://semver.org/).
-
-#### Versioning guidelines
-
-- A typical pull request with a new feature bumps the minor version.
-- Bug fix, documentation change and operational code (i.e. devops) change bump the patch version.
-- A breaking change bumps the major version. Please consult with the Presidio team prior to opening such PRs.
+-   A Github issue suggesting the change should be opened prior to a PR.
```

Comment:
just curious - why was the extra whitespace after the dash added, in this list and next. It doesn't make a difference visually in github, but it deviates from standard markdown syntax.

Reply:
auto indent with vscode